__tests__/app.test.js

```diff
@@ -117,4 +117,53 @@ describe("GET: /api/users", () => {
       .expect(404)
     })
 })
-      
\ No newline at end of file
+
+describe('6. PATCH /api/articles/:article_id  ', () => {
+test("200: responds with updated votes", () => {
+  const newVote = {
+    inc_vote: 3
+  }
+  return request(app)
+    .patch("/api/articles/6")
+    .send(newVote)
+    .expect(200)
+    .then((response) => {
+      const {
+        body: { article },
+      } = response;
+      expect(article).toEqual(
+        {"article_id": 6, "author": "icellusedkars", "body": "Delicious tin of cat food", "created_at": "2020-10-18T01:00:00.000Z", "title": "A", "topic": "mitch", "votes": 3}
+      )
+  })
+ })
+
+ test("400: responds with invalid input for a bad request", () => {
+  const newVote = {
+    inc_vote: 3
+  }
+  return request(app)
+    .patch("/api/articles/%")
+    .send(newVote)
+    .expect(400)
+ })
+
+ test("404: responds with not found error if the article doesn't exist but the entry is valid", () => {
+  const newVote = {
+    inc_vote: 3
+  }
+  return request(app)
+    .patch("/api/articles/1000")
+    .send(newVote)
+    .expect(404)
+ })
+
+ test("500: responds with server error if the newVote does not contain the key inc_vote", () => {
```

Comment:
This shouldn't be an internal server error, this is a bad request so would be a 400 error